TypeSystem/Schema/smoke_test_struct.h

```diff
@@ -20,6 +21,7 @@ CURRENT_STRUCT(Primitives) {
   CURRENT_FIELD(m, bool);
   CURRENT_FIELD(n, std::chrono::microseconds);
   CURRENT_FIELD(o, std::chrono::milliseconds);
+  CURRENT_FIELD_DESCRIPTION(b, "Field descriptions can be set in any order.");
```

Comment:
Let me see how plausible this would be. Keep in mind, we have F# and Markdown export formats, where C++-grade escaping is probably not the wisest choice.